DataDog/storedog-frontend
# WEB-2843 | Add XSS example

## Description
Jira Ticket: https://datadoghq.atlassian.net/browse/WEB-2843

This adds the frontend portion to test the security monitoring of a cross site scripting(xss) vulnerability from the security product.

## How to test
- Pull this branch
- Pull the backend branch from this [PR](https://github.com/DataDog/storedog-backend/pull/26)
- Spin up the backend using `docker compose --csrf up`
- Spin up the frontend `docker compose up`
- Go into the `featureFlags.config.json` file in the site folder
  - update active to be true or false
  - reload the page (hot reload seems to work but if it doesn't show/disappear then refresh)
- On the homepage in the nav you should see an option to input your email, this will have a few testing steps
  - click submit with no input, you should get a validation error
  - enter anything into the input (it being an email isn't important) and submit
  - You should get a thank you message with the input you entered at the end

## Checklist

Before you move on, make sure that:
